js/dataTables.main.js

```diff
@@ -78,4 +78,29 @@ function getMultipleSelectionParams(listId)
 	});
 
 	return oRes;
-}
\ No newline at end of file
+}
+
+/**
+ * Return column JSON declaration for row actions.
+ * Could be part of column or columnDefs declaration of datatable.js.
+ *
+ * @param sTableId
+ * @param iColumnTargetIndex
+ * @returns {*}
+ */
+function getRowActionsColumnDefinition(sTableId, iColumnTargetIndex = -1)
+{
+	let aColumn = {
+		type: "html",
+		orderable: false,
+		render: function ( data, type, row, meta ) {
+			return $(`#${sTableId}_actions_buttons_template`).html();
+		}
+	};
+
+	if(iColumnTargetIndex != -1){
```

Comment:
```suggestion
	if (iColumnTargetIndex !== -1) {
```

We rather have strict type comparisons unless it is known that the data type can vary (eg. can be either `'1'` or `1` depending on the consumer).
This avoid future regression like the one we had on _template-base_ for instance.